extensions/amp-pinterest/0.1/pin-widget.js

```diff
@@ -188,6 +200,7 @@ export class PinWidget {
           'img': {
             'className': '-amp-pinterest-embed-pin-text-icon-attrib',
             'src': pin['attribution']['provider_icon_url'],
+            'alt': 'Pinterest image attribution icon',
```

Comment:
Hmm, does this really provide the right (or useful) context to a screen reader user? For example, if this was `aria-hidden` or had an empty `alt`, the content of the attribution would be `by <author name>`, which would I think would get the point across.

One thing to keep an eye out for on a11y is to make sure we don't spam the user with too much info. For a sighted user, it is easy enough to gloss over the icon, but a screen reader user will have to move past it and at least partially listen to it to see if it is useful.

It seems by omitting this, we would probably make the overall experience better.

Reply:
Based on their JSON format: 
```
attribution: {title: "Next stop: Pinterest!", url: "http://www.flickr.com/photos/kentbrew/6851755809/",…}
author_name: "kentbrew"
author_url: "http://www.flickr.com/photos/kentbrew/"
provider_favicon_url: "https://s.pinimg.com/images/api/attrib/flickr.png"
provider_icon_url: "https://s.pinimg.com/images/api/attrib/flickr@2x.png"
provider_name: "flickr"
title: "Next stop: Pinterest!"
url: "http://www.flickr.com/photos/kentbrew/6851755809/"
```

It does look like they have a `provider_name` attribute that we can use to provide some useful information. Changed to include that.